pkg/apis/application/v1alpha1/types.go

```diff
@@ -896,6 +900,35 @@ type Repository struct {
 	Insecure bool `json:"insecure,omitempty" protobuf:"bytes,7,opt,name=insecure"`
 }
 
+func getCAPath(repoURL string) string {
+	if git.IsHTTPSURL(repoURL) {
+		if parsedURL, err := url.Parse(repoURL); err == nil {
+			if caPath, err := cert.GetCertBundlePathForRepository(parsedURL.Host); err != nil {
```

Comment:
We don't care or log about this error?

Reply:
good point. Warning should help troubleshoting misconfigured cert. Fixed